compiler/rustc_codegen_ssa/src/back/link.rs

```diff
@@ -152,6 +164,17 @@ pub fn link_binary<'a>(
                     );
                 }
             }
+
+            if output == OutFileName::Stdout {
```

Comment:
this as well

Reply:
Sorry I don't get it. Do you mean including this error check in a `OutFileName::some_function`?